app/components/login_button_component.rb

```diff
@@ -0,0 +1,30 @@
+class LoginButtonComponent < ButtonComponent
+  APP_NAME = 'Login.gov'
```

Comment:
We shouldn't need to specify this constant, I'd expect it'd already be available ([related code](https://github.com/18F/identity-idp/blob/8371312258693c0ba4817b4d6498b17318ad0a44/config/application.rb#L24)).

```suggestion
```